### ISSUE TITLE
Not able to find the JS files

### ISSUE THREAD

**Issue by thomasboulongne** [Anthelmed/2021-cours-bachelor-dev-gobelins#1]
It might be an issue related to the Git versioning but I'm unable to find the JS files in the repo. As a developer from Gobelins I believe there *has* to be JS files in a course for Gobelins students.

**Comment by Anthelmed** [comment]
You're right, my apologies, I've added a gh-pages branch with a webgl build of this project. ([direct link](https://anthelmed.github.io/2021-cours-bachelor-dev-gobelins/))
Hoping this is all you need.
